.tx/config

```diff
@@ -0,0 +1,9 @@
+[main]
```

Comment:
Thanks @OsauravO .
We already have transfex setuo.
Can you discard this file so that we can merge in these changes ??